cal-itp/benefits
# Feat: Cal-ITP staff edit

Closes #2200 

### Testing locally
- Set up Google SSO for the admin interface - see [docs](https://docs.calitp.org/benefits/configuration/admin-interface/#environment-variables)
- Set `GOOGLE_SSO_STAFF_LIST` to your email
   - Make sure your email is not in `GOOGLE_SSO_SUPERUSER_LIST`
- `F5` to start the app
- Go to `/admin` and log in with your Google account
- Navigate around as a staff user

### Screenshots

This is what the user sees upon logging in:

![image](https://github.com/user-attachments/assets/7610a9bc-ab6e-4a85-b27d-37a07497069a)

When viewing model objects, the user can edit fields that are not in the `readonly_fields` list or `exclude` list for each model.

Here is an example using the `TransitAgency` model:


![image](https://github.com/user-attachments/assets/064b4ab0-3459-4305-9ad5-f470ddc82479)
